tests/system/test_export.py

```diff
@@ -27,7 +28,7 @@ def test_export_config(self):
         """
         Test export default config
         """
-        config = yaml.load(self.command_output)
+        config = yaml.load(self.command_output, Loader=Loader)
```

Comment:
What caused this change? Did the PyYAML version get updated?

Reply:
yes it changed from `4.2` to `5.3`, which led to a warning when not specifying the `Loader`.